src/main/java/org/commonjava/util/sidecar/util/ProxyStreamingOutput.java

```diff
@@ -72,26 +155,4 @@ public void write( OutputStream output ) throws IOException
             }
         }
     }
-
-    private void closeBodyStream( InputStream is )
-    {
-        if ( is == null )
-        {
-            return;
-        }
-
-        try
-        {
-            is.close();
-        }
-        catch ( IOException e )
-        {
-            if ( otel.enabled() )
```

Comment:
Is this possible to happen in the new okhttp3 way?

Reply:
@ligangty it's verified, IOException is never thrown in responsebody.close().